docker/1.2-1/base/Dockerfile.cpu

```diff
@@ -89,10 +89,10 @@ RUN echo "conda ${CONDA_PKG_VERSION}" >> /miniconda3/conda-meta/pinned && \
     conda config --system --set auto_update_conda false && \
     conda config --system --set show_channel_urls true && \
     echo "python ${PYTHON_VERSION}.*" >> /miniconda3/conda-meta/pinned && \
-    conda install -c conda-forge python=${PYTHON_VERSION} && \
-    conda install conda=${CONDA_PKG_VERSION} && \
+    conda install -c conda-forge python=${PYTHON_VERSION} --solver classic && \
+    conda install conda=${CONDA_PKG_VERSION} --solver classic && \
```

Comment:
just curious, what value does --solver classic add?

Reply:
The install command failed without it. The error message said to do that.

Reply:
newer conda=>23.10 have libmamba as default solver
https://conda.github.io/conda-libmamba-solver/user-guide/
https://conda.github.io/conda-libmamba-solver/user-guide/libmamba-vs-classic/

my error was 
```
35.21 Error while loading conda entry point: conda-libmamba-solver (libarchive.so.20: cannot open shared object file: No such file or directory)
35.31
35.31 CondaValueError: You have chosen a non-default solver backend (libmamba) but it was not recognized. Choose one of: classic
35.31
``` 

you could either use `--solver classic` or forcing all channels to use conda-forge
```shell
--- a/docker/1.2-1/base/Dockerfile.cpu
+++ b/docker/1.2-1/base/Dockerfile.cpu
@@ -88,11 +88,13 @@ RUN echo "conda ${CONDA_PKG_VERSION}" >> /miniconda3/conda-meta/pinned && \
     # Conda configuration see https://conda.io/projects/conda/en/latest/configuration.html
     conda config --system --set auto_update_conda false && \
     conda config --system --set show_channel_urls true && \
+    conda config --add channels conda-forge && \
     echo "python ${PYTHON_VERSION}.*" >> /miniconda3/conda-meta/pinned && \
-    conda install -c conda-forge python=${PYTHON_VERSION} --solver classic && \
-    conda install conda=${CONDA_PKG_VERSION} --solver classic && \
+    conda update --all --override-channels -c conda-forge -n base && \
+    conda install -c conda-forge python=${PYTHON_VERSION} && \
+    conda install conda=${CONDA_PKG_VERSION} && \
     conda update -y conda && \
-    conda install -c conda-forge pyarrow=${PYARROW_VERSION} --solver classic && \
+    conda install -c conda-forge pyarrow=${PYARROW_VERSION} && \
     conda install pip --force-reinstall && \
     python3 -m pip install --upgrade pip && \
     python3 -m pip install pyOpenSSL==23.1.0 && \
```